pkg/cmd/pulumi/packagecmd/project_sdk_link.go

```diff
@@ -371,10 +371,14 @@ func linkPythonPackage(ws pkgWorkspace.Context, root string, pkg *schema.Package
 		importName = strings.ReplaceAll(pkg.Name, "-", "_")
 	}
 
+	namespace := "pulumi"
+	if pkg.Namespace != "" {
+		namespace = strings.ReplaceAll(pkg.Namespace, "-", "_")
+	}
 	fmt.Println()
 	fmt.Println("You can then import the SDK in your Python code with:")
 	fmt.Println()
-	fmt.Printf("  import pulumi_%s as %s\n", importName, importName)
+	fmt.Printf("  import %s_%s as %s\n", namespace, importName, importName)
```

Comment:
Feels like this should use the pyPack function as well?

Reply:
I avoided it here because it's private to codegen currently, and the cross dependency felt a little weird.  But no strong opinion, happy to change that.

Reply:
Whoops, while doing this I realized that the code above had a bug when the packagename was set in packageinfo (The bug was already preexisting, but it's not fixed).

Reply:
> and the cross dependency felt a little weird

Nah, this stuff is basically an extension of programgen it should be calling into codegen code.

Reply:
Makes sense, it's done that way now :)